app/views/hyrax/collections/_show_parent_collections.html.erb

```diff
@@ -0,0 +1,30 @@
+<% if @parent_collections.nil? || @parent_collections.empty? %>
+		<div class="alert alert-warning" role="alert"><%= t('hyrax.collections.show.no_visible_parent_collections') %></div>
+<% else %>
+		<div class="media-body">
+			<h4 class="media-heading">
+				<div class="less-parent-collections-block" id="less-parent-collections">
+					<% i = 0 %>
+					<% include_more_tag = false %>
+					<% @parent_collections.each do |document| %>
+							<ul>
+								<li>
+									<%= link_to document, [hyrax, document], id: "src_copy_link_#{document.id}" %>
+								</li>
+							</ul>
+							<% i += 1 %>
+							<% include_more_tag = true if(i == @presenter.parents_to_show && i < @parent_collections.size) %>
+							<% if include_more_tag %>
+									<button id="show-more-parent-collections" class="btn show-more-parent-collections-btn"><%= t("hyrax.collections.show.show_more_parent_collections") %></button>
+									</div>
```

Comment:
I'd verify that the markup properly closes these `<div>` tags.  Generally you'd probably want to keep paired opening and closing `<div>` tags within the same `if/else` block.

Reply:
Improved presenter to include a variable for the visible_parent_collections and more_parent_collections.